lxd/db/images.go

```diff
@@ -76,7 +76,7 @@ func ImagesGetExpired(db *sql.DB, expiry int64) ([]string, error) {
 		results = append(results, r[0].(string))
 	}
 
-	return []string{}, nil
```

Comment:
Oh wow...

Reply:
@brauner this went unnoticed despite the review (#3817), unit tests! :)

Reply:
I was trying to find this bug in issue #3904, but I found a problem with the unit/integration test instead :)